habitat-baselines/habitat_baselines/agents/ppo_agents.py

```diff
@@ -7,38 +7,42 @@
 
 import argparse
 import random
+from dataclasses import dataclass
 from typing import Dict, Optional
 
 import numpy as np
 import torch
 from gym.spaces import Box
 from gym.spaces import Dict as SpaceDict
 from gym.spaces import Discrete
+from omegaconf import DictConfig, OmegaConf
 
 import habitat
-from habitat.config import Config
 from habitat.core.agent import Agent
 from habitat.core.simulator import Observations
 from habitat_baselines.rl.ddppo.policy import PointNavResNetPolicy
 from habitat_baselines.utils.common import batch_obs
 
 
-def get_default_config() -> Config:
-    c = Config()
-    c.INPUT_TYPE = "rgb"
-    c.MODEL_PATH = "data/checkpoints/gibson-rgb-best.pth"
-    c.RESOLUTION = 256
-    c.hidden_size = 512
-    c.RANDOM_SEED = 7
-    c.PTH_GPU_ID = 0
-    c.goal_sensor_uuid = "pointgoal_with_gps_compass"
-    return c
+@dataclass
+class PPOAgentConfig:
+    INPUT_TYPE: str = "rgb"
```

Comment:
Do we still want to keep it uppercase here?

Reply:
I am the one that left it on purpose upper case. This is because there are some fields in the dataclass that are the same as the habitat_baselines config and I wanted to make sure users would know it is different. (I got confused before)